XCUITest/TPSidebarBadge.swift

```diff
@@ -94,32 +106,38 @@ class TrackingProtectionSidebar: BaseTestCase {
         waitforExistence(element: app.staticTexts["Tracking Protection"])
 
         // Disable tracking protection
-        waitforExistence(element: app.switches["TrackingProtectionToggleView.toggleTrackingProtection"])
-        app.switches["TrackingProtectionToggleView.toggleTrackingProtection"].tap()
+        waitforExistence(element: app.switches["Tracking Protection.Toggle"])
+        app.switches["Tracking Protection.Toggle"].tap()
 
         // Reopen the tracking protection sidebar
         app.otherElements["URLBar.trackingProtectionIcon"].tap()
 
         // Wait for the sidebar to open
         waitforExistence(element: app.staticTexts["Tracking Protection"])
 
-        // Check for the tracker count in the sidebar
-        let counters = app.staticTexts.matching(identifier: "TrackingProtectionBreakdownItem.counterLabel")
-
-        for i in 0..<counters.staticTexts.count {
-            XCTAssertEqual(counters.element(boundBy: i).label, "--")
-        }
-
-        // Close the sidebar
-        app.buttons["TrackingProtectionView.closeButton"].tap()
-        waitforNoExistence(element: app.staticTexts["TrackingProtectionToggleView.toggleTrackingProtection"])
-
-        // Check that no counter exists (tracking protection: disabled)
-        XCTAssertEqual(app.staticTexts["TrackingProtectionBadge.counterLabel"].exists, false)
-
+        // Check for all 0 tracker count values in menu
```

Comment:
I'd recommend moving this into a function since it's used a couple times. Something like waitForZeroTrackersBlocked